### DISCUSSION TITLE
Using configuration values in Adonis 5

### DISCUSSION THREAD

**Discussion by zmaglica** [adonisjs/core#1275] [Help] [answered]
Hi.

How can I use custom configuration values that are placed inside config folder? In Adonis v4 there was a Config Provider and I can't find anything similar in adonis v5.

**Comment by thetutlage** [comment, marked as answer]
There is still the `ConfigProvider` (but yes not documented).

```ts
import Config from '@ioc:Adonis/Core/Config'
```

or you can also simply import the config files 

```ts
import app from './relative/path/to/config/app'
```

The later one will give you intellisense support

**Reply by zmaglica** [reply]
Thank you. I manage to fetch config values using `ConfigProvider`.
